project/vision_backend/tasks.py

```diff
@@ -285,14 +285,18 @@ def collect_all_jobs():
 def _handle_job_result(job_res: JobReturnMsg):
     """Handles the job results found in queue. """
 
+    task_name = job_res.original_job.task_name
+
     if not job_res.ok:
         logger.error("Job failed: {}".format(job_res.error_message))
         mail_admins("Spacer job failed", repr(job_res))
```

Comment:
I wonder if we should skip the `mail_admin` step for the errors we handle? Just to reduce noise?

Reply:
Probably, yeah - at least for the time being.

If we can later resolve the job order (or something else) so that this AssertionError doesn't happen nearly as often, then it'd be good to turn these admin mails back on, so we can be notified about other types of train/deploy errors.